### ISSUE TITLE
File size ADLB

### ISSUE THREAD

**Issue by martingebel** [SafetyGraphics/safetyGraphics#661]
Dear All,

the tool actually looks amazing. Great work!

Therefore, I tried to use it for a 800 patient study. Eve n though I restricted the lab data set to only the relevant variables (8) and measurements (5) I always get the error message "maximum upload size exceeded".

Is it possible to increase the allowed file size?

Thanks in advance

**Comment by martingebel** [comment]
Great, upload works smoothly now . Thanks a lot for covering this so quickly.

Nevertheless, I am only able use the lb chart when just uploading the ADLB. When I additionally upload ADSL I receive error messaged
![error](https://user-images.githubusercontent.com/91063474/157430876-964d05ce-15d8-4391-a37b-8136db6b3246.PNG)


Also receiving error messages when just uploading ADSL (hatever variable I use as Treatment group)
![error](https://user-images.githubusercontent.com/91063474/157431160-13664695-a243-4191-b6b3-060ab7cb7876.PNG)

**Comment by jwildfire** [comment]
Yeah, the 2nd error is a [known bug](https://github.com/SafetyGraphics/safetyCharts/issues/79) that has been fixed and will be included in the  upcoming v2.1 release on CRAN. For now, you can try running installing the safetyCharts dependency from github as a work around: `devtools::install_github('safetygraphics\safetyCharts', ref='dev')`

Not sure what is going on with the first error block. Are you using the `safetyGraphicsInit()` process? That part of the app hasn't had tons of testing, so it's entirely possible you've found a new issue ...

**Comment by martingebel** [comment]
You are right, loading the data into R using the `safetyGraphics::safetyGraphicsApp(domainData=list(labs=Labs,dm=adsl))` it works fine. So, it seems that the `safetyGraphicsInit()` function is causing the issue when using two of my data sets.

Also the updated version from Github solved my second error. Thanks

**Comment by jwildfire** [comment]
Good deal, @martingebel. Going to go ahead and close out this issue. Don't hesitate to reach out with more questions/feedback.

**Comment by paulinebaur** [comment]
Dear All,

just like @martingebel, I tried to use `safetyGraphicsInit()` (with safetyGraphics version 2.1.0) and I ran into the same error message (arguments imply differing number of rows: 1,0). I encountered this error with some of my own data as well as after saving  `safetyData::adam_adlbc`, `safetyData::adam_adae` and `safetyData::adam_adsl` as `.csv` files and uploading these files to `safetyGraphicsInit()`. 

Although everything works perfectly using the command `safetyGraphicsApp()`, I would prefer working with `safetyGraphicsInit()`, since it gives researchers, who don’t use R the chance to upload and analyze their data. 
 
Therefore, I wanted to ask, if you know a workaround around this error or have some ideas for me on how to solve my problem.

Thanks for your help in advance!

**Comment by jwildfire** [comment]
Interesting, kind of forgot about that hack that I put in to get it to render consistently. 

@paulinebaur could you try starting the app with `safetyGraphicsInit(delayTime=5000)` and see if that helps. This tells the app to wait for 5 seconds (instead of the default 1 second) before rendering. If that workaround does the trick, we can try to dig in and find a more permanent fix as time allows.

**Comment by paulinebaur** [comment]
Thank you for your answers. Indeed increasing the `delayTime` helps and `safetyGraphicsInit()` works just fine now. Thanks a lot!
